WebDriverAgentLib/Categories/XCUIElement+FBScrolling.m

```diff
@@ -87,13 +88,27 @@ - (BOOL)fb_scrollToVisibleWithNormalizedScrollDistance:(CGFloat)normalizedScroll
   if (self.fb_isVisible) {
     return YES;
   }
+
   NSArray *possibleParents = @[
                                @(XCUIElementTypeScrollView),
                                @(XCUIElementTypeCollectionView),
                                @(XCUIElementTypeTable),
-                              ];
+                               ];
     
-  XCElementSnapshot *scrollView = [self.lastSnapshot fb_parentMatchingOneOfTypes:possibleParents];
+    XCElementSnapshot *scrollView = [self.lastSnapshot fb_findVisibleParentMatchingOneOfTypesWithFilter:possibleParents
+      filter:^(XCElementSnapshot *snapshot) {
+          if ([snapshot isWDVisible] && [snapshot fb_hasMoreThanOneVisibleChildSnapshot]) {
+            return YES;
+          }
+          return NO;
```

Comment:
But both are methods return `BOOL` and `&&` will return `BOOL`, so it should be fine